docs/project/os-onboarding.md

```diff
@@ -0,0 +1,105 @@
+
+There are multiple reasons to add a new OS reference to a release branch:
+
+- Known product (as opposed to test) breaks that require validation and regression testing.
+- Past experience suggests that coverage is required to protect against risk.
+- OS version is or [will soon go EOL](https://github.com/dotnet/runtime/issues/111818#issuecomment-2613642202) and should be replaced by a newer version.
+
+For example, we frequently need to backport Alpine updates to release branches to avoid EOL references but less commonly for Ubuntu, given the vast difference in support length.
+
+A good strategy is to keep `main` at the bleeding edge of new OS versions. That way those references have a decent chance of never needing remediation once they end up in release branches. Being _active_ in `main` enables being _lazy_ in `release/`.
+
+### Updating or removing coverage
+
+We will often replace an older OS version with a new one, when it comes available. This approach is an effective strategy of maintaining the same level of coverage and of remediating EOL OSes ahead of time. For the most part, we don't need to care about a specific version. We just want coverage for the OS, like Alpine.
+
+We should remediate any EOL OS references in our codebase. They don't serve any benefit and come with some risk.
+
+In the case that a .NET version will be EOL in <6 (and certainly <3) months, new coverage can typically be skipped. We may even be able to skip remediating EOL OS references. We often opt to stop updating [Supported OSes](https://github.com/dotnet/core/blob/main/os-lifecycle-policy.md) late in support period for related reasons. A lazy approach is often the best approach late in the game. Don't upset what's working.
+
+## Building
+
+Our [build methodology](https://github.com/dotnet/runtime/blob/main/docs/project/linux-build-methodology.md) is oriented around cross-compiling, enabling us to target an old OS version and run on newer ones. It is uncommon for us to need to make changes to the build to address new OS versions, however, there are [rare cases where we need to make adjustments](https://github.com/dotnet/runtime/issues/101944).
+
+We use both containers and VMs for building, depending on the OS. If we test in a container, we likely build in a container. Same for VMs.
+
+Our primary concern is ensuring that we are using [supported operating systems and tools for our build](https://github.com/dotnet/runtime/tree/main/docs/workflow/requirements).
+
+Our Linux build containers are based on Azure Linux. We [typically need to update them](https://github.com/dotnet/runtime/issues/112191) with a new version of Azure Linux once per release. We do not update the toolset, however. That's fixed, per release.
+
+For Apple, we likely need to make an adjustment at each macOS or iOS release to account for an XCode version no longer being supported.

```

Comment:
```suggestion
For Apple, we likely need to make an adjustment at each macOS or iOS release to account for an Xcode version no longer being supported.

```

Reply:
I see we have this casing mistake in several other places under docs. It would be nice to fix it while you are on it.